test/stats/test_distribution_exponential.fypp

```diff
@@ -72,46 +72,46 @@ contains
 
     #:for k1, t1 in RC_KINDS_TYPES
     subroutine test_expon_rvs_${t1[0]}$${k1}$
-        ${t1}$ :: res(10), scale
+        ${t1}$ :: res(10), loc, scale
         integer, parameter :: k = 5
         integer :: i
         integer :: seed, get
         #:if t1[0] == "r"
             #! for real type
         ${t1}$, parameter :: ans(10) =                                         &
-                            [0.609680481289574416337018192280083895_${k1}$,    &
-                             0.137541023585612635452927558314210417_${k1}$,    &
-                             0.134921508232253721063879462841820585_${k1}$,    &
-                              1.33766060689229752493171569464417802_${k1}$,    &
-                             0.111148487576340881943792737729381770_${k1}$,    &
-                             0.533951653963536868966836361020492979_${k1}$,    &
-                              1.96897428558727671799033487332053483_${k1}$,    &
-                             0.371111977992924465160247867364281152_${k1}$,    &
-                             0.811918715695663687862785688290993341_${k1}$,    &
-                             0.404637854946697868759504975362991277_${k1}$]
+                            [ 1.37178108290154243675829093263018876_${k1}$,    &
```

Comment:
Ideally, these values should not be modified. Only the calls should be modified and return the same values as the previous versions.